velox/exec/tests/OutputBufferManagerTest.cpp

```diff
@@ -971,6 +977,60 @@ TEST_P(AllOutputBufferManagerTest, outputBufferUtilization) {
   verifyOutputBuffer(task, OutputBufferStatus::kFinished);
 }
 
+TEST_P(AllOutputBufferManagerTest, outputBufferStats) {
+  const vector_size_t size = 100;
+  const std::string taskId = std::to_string(rand());
+  initializeTask(taskId, rowType_, kind_, 1, 1);
+  // check the stats kind
+  ASSERT_EQ(getStats(taskId).kind, kind_);
+
+  const int pageNum = 3;
+  int totalSize = 0;
+  for (int pageId = 0; pageId < pageNum; pageId++) {
+    // enqueue pages and check stats of buffered data
+    enqueue(taskId, 0, rowType_, size);
+    totalSize += size;
+    // force ArbitraryBuffer to load data, otherwise the data would
+    // not be buffered in DestinationBuffer
+    if (kind_ == PartitionedOutputNode::Kind::kArbitrary) {
+      fetchOne(taskId, 0, pageId);
+    }
+    OutputBufferStats statsEnqueue = getStats(taskId);
+    ASSERT_EQ(statsEnqueue.destinationBuffers[0].pagesBuffered, 1);
+    ASSERT_EQ(statsEnqueue.destinationBuffers[0].rowsBuffered, size);
+
+    // ack pages and check stats of sent data
+    fetchOneAndAck(taskId, 0, pageId);
+    OutputBufferStats statsAck = getStats(taskId);
+    ASSERT_EQ(statsAck.destinationBuffers[0].pagesSent, pageId + 1);
+    ASSERT_EQ(statsAck.destinationBuffers[0].rowsSent, totalSize);
+    ASSERT_EQ(statsAck.destinationBuffers[0].pagesBuffered, 0);
+    ASSERT_EQ(statsAck.destinationBuffers[0].rowsBuffered, 0);
+  }
+
+  // set outputBuffer to NoMoreBuffers and check stats
```

Comment:
nit: Make sure comments are full sentences. Start with a capital letter and end with a period.

Reply:
quick fixed.